spec/models/post_spec.rb

```diff
@@ -2238,6 +2238,17 @@ def updates_topic_updated_at
     end
   end
 
+  describe "full_url" do
```

Comment:
I was going to update the post.url method but realised that only Topics have this issue, decided to add a test here for subfolders since we don't have any coverage for full_url.